spec/README.md

```diff
@@ -123,3 +123,30 @@ All other names in these categories are reserved for the use of implementations
 > [!NOTE]
 > Users defining custom names SHOULD include at least one character outside these ranges
 > to ensure that they will be compatible with future versions of this specification.
+
+Later versions of this specification will not introduce changes
+to the data model that would result in a data model representation
+based on this version being invalid.
+
+> For example, existing interfaces or fields will not be removed.
+
+Later versions of this specification MAY introduce changes
+to the data model that would result in future data model representations
+not being valid for implementations of this version of the data model.
+
+> For example, a future version could introduce a new keyword,
+> whose data model representation would be a new interface
+> that is not recognized by this version's data model.
+
+Later specification versions will not introduce syntax that cannot be
+processed by this version of the data model.
+However, the results of such processing might not be the same as that produced
+but future version of the data model.
```

Comment:
```suggestion
However, a future version of the specification could change the meaning
of a particular data model.
```

I'm not sure if this is exactly what you meant, but "the results of such processing" confused me. I think you're trying to say that future changes to the spec may change the semantics of a particular data model instance, but will never expand the set of valid data model instances?

Reply:
I think it may actually be clearer to leave out this sentence, especially as the following note clarifies the meaning sufficiently.
```suggestion
```